frontend/src/components/dashboard/Dashboard.js

```diff
@@ -288,11 +288,10 @@ export default function Dashboard() {
                   <DescLinks name="Description Links"/>
                 </Container>
               </Route>
-              <Route exact path="/picture-id">
-                <Container className={classes.withPadding}>
-                  <ArtworkCloseUpCard/>
-                </Container>
-              </Route>
+              <Route
+                exact path="/gallery/:id"
```

Comment:
By using the component prop of Route now the `id` will pass as a prop to ArtworkCloseUpCard. The `:id` part of the string will have the actual `id`.